omise/setting.php

```diff
@@ -0,0 +1,113 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class Setting
+{
+    protected $submit_action = 'omise_save_setting';
```

Comment:
@nimid continue from https://github.com/omise/omise-prestashop/pull/10#discussion_r95589021 (not sure why the comment doesn't show here 🤔 )

I just found that if we use `const SUBMIT_ACTION = 'omise_save_setting';`
we can get rid of `Setting::getSubmitAction()` method from everywhere (at omise.php & OmiseTest.php).